README.md

```diff
@@ -123,6 +123,28 @@ Or shorten the loop as one-liner using the element & attribute selector:
 print(query['@m', ('id', 'scores__cosine')])
 ```
 
+### Query
+
+You can get specific document by its id:
+
+```python
+doc = ann.get_doc_by_id('<doc_id>')
+```
+
+And you can also get the documents with `limit` and `offset`, which is useful for pagination:
+
+```python
+docs = ann.get_docs(limit=10, offset=0)
+```
+
+Furthermore, you can also get the documents ordered by a specific column from the index:
+
+```python
+docs = ann.get_docs(limit=10, offset=0, order_by='x', ascending=True)
```

Comment:
we should tell users to put `x` into `filterable_attrs` when initializing `annlite` if they want to use this feature

Reply:
I have added the note message below.